hmpy/views/switch.py

```diff
@@ -0,0 +1,118 @@
+"""This module defines the SwitchView class."""
```

Comment:
Adding comments from the PR to the review:

Before I just to deep into the actual code a couple of suggestions right off the bat.
First off, the button should be centered vertically. Additionally I think we should probably provide an optional arg to the constructor for adding a title, otherwise there is no way to really indicate which button does what

Also wondering if there is any specific reason you chose not to implement the switch with some kind of a QAbstractButton?